daemon.py

```diff
@@ -158,10 +158,23 @@ def start(self, foreground=False):
         pid = self.pid()
 
         if pid:
-            message = "pidfile %s already exists. Is it already running?\n"
-            log.error(message % self.pidfile)
-            sys.stderr.write(message % self.pidfile)
-            sys.exit(1)
+            # Check if the pid in the pidfile corresponds to a running process
```

Comment:
let's use psutil.pid_exists, it's moe concise, easier to read and works on Windows (even though this part of the code is not run on Windows) http://pythonhosted.org/psutil/#psutil.pid_exists